.github/workflows/deployDemo.yml

```diff
@@ -50,11 +50,15 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Build deploy artifact
+        env:
+          REACT_APP_BASE_URL: https://demo.simplereport.gov
+          REACT_APP_BACKEND_URL: https://demo.simplereport.gov/api
+          PUBLIC_URL: /app/
         run: yarn run build
       - uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Deploy frontend app
-        run: yarn run deploy-azure-${{ env.DEPLOY_ENV }}
+        run: az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdemoapp --destination-path '/app'
       - name: Purge CDN
-        run: az cdn endpoint purge --resource-group prime-simple-report-${{ env.DEPLOY_ENV }} --profile-name simple-report-${{ env.DEPLOY_ENV }} --name simple-report-${{ env.DEPLOY_ENV }} --content-paths "/app/*" --no-wait
\ No newline at end of file
+        run: az cdn endpoint purge --resource-group prime-simple-report-${{ env.DEPLOY_ENV }} --profile-name simple-report-${{ env.DEPLOY_ENV }} --name simple-report-${{ env.DEPLOY_ENV }} --content-paths "/app/*" --no-wait
```

Comment:
Same question as previous PR, hoisted to here: would it be reasonable to just purge the index.html? I know that some CDNs get irritated if you purge too often, and I thought not needing to was one benefit to versioned static files.

Reply:
I am not sure how to limit this down with out maintain a list of flies that need to be purged (azure does not have exclude path). we would need to purge everything not in the `/static` and also purge `/static/commit.txt` or more likely move it out of the static dir

We might be able to adjust the yarn build command so that files with cache busting are not in the same directory as those without it

Reply:
Might be worth investing some time into why we need to purge in the first place

Reply:
Added a ticket for this: https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/1014